src/lib.rs

```diff
@@ -46,6 +67,200 @@ impl UniffiCustomTypeConverter for Uuid {
     }
 }
 
+pub struct StatusLists;
+
+impl TypedStatusMapProvider<Uri, BitstringStatusListCredential> for StatusLists {
+    async fn get_typed(&self, id: &Uri) -> Result<MaybeCached<StatusList>, ProviderError> {
```

Comment:
I'm surprised `Uri` would work as you haven't defined a wrapper type. Is this code actually used?

Reply:
`w3c_vc_barcodes::verify` requires this implementation, but since we don't have a test w3c-vc-barcode with a valid status list URI we are bypassing the retrieval with this code. 

Adding a TODO for the future.

Reply:
I think I completely missed the fact that it was an trait implementation 🤦